src/overlays/actors/ovl_Dm_Char02/z_dm_char02.h

```diff
@@ -8,12 +8,16 @@ struct DmChar02;
 typedef void (*DmChar02ActionFunc)(struct DmChar02*, PlayState*);
 
 typedef struct DmChar02 {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x14C];
-    /* 0x0290 */ DmChar02ActionFunc actionFunc;
-    /* 0x0294 */ char unk_294[0x60];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ char pad188[0x108];
+    /* 0x290 */ DmChar02ActionFunc actionFunc;
+    /* 0x294 */ char pad294[0x4C];
+    /* 0x2E0 */ s16 animIndex;
+    /* 0x2E2 */ char pad2E2[0xE];
+    /* 0x2F0 */ u32 unk2F0;
 } DmChar02; // size = 0x2F4
 
-extern const ActorInit Dm_Char02_InitVars;
+const ActorInit Dm_Char02_InitVars;
```

Comment:
```suggestion
extern const ActorInit Dm_Char02_InitVars;
```
Should keep the extern.